NEW_RELEASE_NOTES.md

```diff
@@ -8,3 +8,5 @@ appropriate header in [RELEASE_NOTES.md](./RELEASE_NOTES.md).
 
 ## Release notes for next branch cut
 
+- materials: improved size reduction of OpenGL/Metal shaders by ~65% when compiling materials with
+             size optimizations (`matc -S`)
```

Comment:
kind of obvious, but you could add a `[recompile materials]`

Reply:
Sure